libs/tools/barista/src/main.ts

```diff
@@ -121,6 +122,7 @@ async function buildPages(): Promise<void[]> {
 
   const allPages = await Promise.all(files);
   const overviewPages = await overviewBuilder();
+  await uxDecisionGraphGenerator(); // TODO: should this only be built for the internal version?
```

Comment:
I don't know if we only want to build this for the internal version? 
There may be some content in there that should be internal, at least for now. 

Please add the check that this is only built for internal.